src/styles/common/_buttons.scss

```diff
@@ -198,3 +202,15 @@ button svg {
         }
     }
 }
+
+@include breakpoint($medium-screen) {
+    .btn.bdl-Button--responsive {
+        .btn-content {
+            display: none;
+        }
+
+        .bdl-btn-icon {
+            display: block;
+        }
+    }
+}
```

Comment:
Do we know how many buttons in EUA will use these new props and styles?

What are your thoughts on putting the styles in the parent component like this?
```
// ParentButton.scss

.ParentButton {
    .bdl-btn-icon {
        display: none;
    }
}

.is-responsive-web {
    @include breakpoint($medium-screen) {
        .ParentButton {
            .btn-content {
                display: none;
            }

            .bdl-btn-icon {
                display: block;
            }
        }
    }
}
```

This way you wouldn't need to pass additional props to the component.

Reply:
AFAIK only the share button and potentially the create new item button will use these new props and styles, so I think it makes sense to put the styles in parent components.